test/toc-check.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('colors');
+var toc = require('markdown-toc');
+var Diff = require('diff');
+var fs = require("fs");
+var glob = require("glob");
+
+var files = glob.sync("**/*.md", {ignore: ["node_modules/**/*.md"]})
+
+function removeTocHeader(str, ele, arr) {
+  return str.indexOf('Table of Contents') === -1;
+}
+
+function getCurrentToc(str, remove){
+  if(remove === undefined) remove = 0;
+  var startIndex = str.indexOf('Table of Contents') + 19;
+  var endIndex = str.indexOf("## ", startIndex);
+  var result = str.substring(startIndex, endIndex - remove);
+  
+  if(result[result.length -1] == '\n' || result[result.length -1] == ' '){
+    return getCurrentToc(str, remove+1);
```

Comment:
I'm not sure what this recursion is for.

Reply:
It was for removing new lines and blank spaces, I just modified it to make it more readable

Reply:
Could you just call [`String.prototype.trim()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) on the string instead?

Reply:
Done! Sorry, I completely forgot about that :sweat_smile: